worlds/sm64ex/Options.py

```diff
@@ -127,6 +127,33 @@ class MoveRandomizerActions(OptionSet):
     valid_keys = [action for action in action_item_table if action != 'Double Jump']
     default = valid_keys
 
+sm64_options_groups = [
+    OptionGroup("Logic Options", [
+        AreaRandomizer,
+        BuddyChecks,
+        ExclamationBoxes,
+        ProgressiveKeys,
+        EnableCoinStars,
+        EnableMoveRandomizer,
+        MoveRandomizerActions,
```

Comment:
These two options cause the options page to render awkwardly. Since they are related and would take up enough room, they could probably go into their own category.

Reply:
Added. Also moved `StrictMoveRequirements` into this new grouping.